### PR TITLE
Update FAQ with questions from AngusG

### PR DESCRIPTION
**Pull request by nottombrown**

### CODE REVIEW

File: contest_proposal.md [hunk 1]
```diff
@@ -364,3 +364,17 @@ All this considered, for a contest with N models, we could imagine having maybe
...
+
+- If the model outputs "bird", the true class label must be bird.
+- If the model outputs "bicycle", the true class label must be bicycle.
+- If the model outputs "abstain", the true class label can be either bird or bicycle.
+- The model must abstain on no more than 20% of the clean, un-modified training data.
+
+
+### Are confident mistakes on the clean eligibility dataset acceptable provided the model achieves at least 80% accuracy?
+
+Confident mistakes on the clean eligibility dataset are acceptable. However, defenses that make confident mistakes on the clean eligibility dataset are highly likely to make confident mistakes on adversarial images as well.
```
Comment (carlini) [inline contest_proposal.md]: It's sort of a tautology, no? An attacker could just upload the image from the clean dataset the model got wrong.

Comment (nottombrown) [reply contest_proposal.md]: I mostly agree with you. The clean eligibility dataset is held-out however, and I'd like to minimize the amount of information that we leak about the contents of the eligibility set.

Comment (nottombrown) [reply contest_proposal.md]: Merging this for now. Feel free to reopen this if you think we should change the policy.

Comment (carlini) [reply contest_proposal.md]: Ah I see, yeah that makes sense.

**Pull request merged**